### PR TITLE
P1/feat/7 carousel

### PR DESCRIPTION
**Pull request by netlightebba**
There is a lot in this PR. The functionality of the carousel and kanban combined, where the user can choose an action from the carousel and it appears in the kanban.  The user can also regret an accepted action (but not performed) and mark an action as performed by moving it in kanban. The functionality is connected to the database and writes the changes to the DB. 

The carousel has a behaviour that is not perfect but that will be fixed in a later stage. 
@mabackstr please help us approve this PR when our team is on vacay, come and ask if you have questions! :)

### CODE REVIEW

File: app/assets/main/react_components/User/CarouselActionItem.jsx [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+
+  const gridStyle = {
+    height: "350px",
+    gridTemplateRows: "25px 40px 200px 60px",
+  };
+
+  return (
+    <div
+      className="callout shadow-none m-2 p-2 grid grid-cols-1 grid-rows-4 gap-1"
+      style={gridStyle}
```
Comment (mabackstr) [inline app/assets/main/react_components/User/CarouselActionItem.jsx:44]: Har kanske tailwind css något att erbjuda för ert grid? För enkelhetens skull (men ni kanske redan undersökt detta).

File: app/assets/main/react_components/User/CarouselHeader.jsx [hunk 2]
```diff
@@ -0,0 +1,12 @@
+import React from "react";
+
+function CarouselHeader() {
```
Comment (mabackstr) [inline app/assets/main/react_components/User/CarouselHeader.jsx:3]: Kanske uppdatera för att hålla konsekvent för samtliga?

```suggestion
const CarouselHeader = () => {
```

**Review by mabackstr — APPROVED** [review]
Great job!

**Review by netlight-saraandersson — APPROVED** [review]
Superbra jobbat! :D

**Pull request merged**